libraries/client/client_webhook.py

```diff
@@ -88,6 +88,7 @@ def process_webhook(self):
             'resource_type': rc.resource.type,
             'title': rc.resource.title,
             'manifest': json.dumps(rc.as_dict()),
+            'last_updated': datetime.utcnow
```

Comment:
You're giving it a function reference here, rather than actually calling utcnow(). This is killing all the client_webhook tests.